config/install.yaml

```diff
@@ -1398,6 +1398,7 @@ apiVersion: v1
 data:
   config.yaml: |-
     logLevel: 3
+    defaultUpgradeStrategy: ""
```

Comment:
this replaces the `dataLossPrevention` feature flag, and has 3 possible states: 

- "" means no op
- "pause-and-drain"
- "progressive"